### PR TITLE
Fix Places formatting

### PR DESCRIPTION
**Pull request by PravinPK**
- Added release notes (subfolder and main page)
- Fix the API reference formatting issue
- Fix API header

### CODE REVIEW

File: src/pages/documentation/release-notes/index.md [hunk 1]
```diff
@@ -110,6 +110,10 @@ Note that all these extensions must be updated together. Flutter and React plugi
 
 * Major version update for [Adobe Journey Optimizer](../adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
 
+### Android Places 2.0.0
+
+* Major version update for [Adobe Experience Platform Location Service](../places/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-places-android).
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:115]: already -> now

File: src/pages/documentation/places/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Release Notes
+
+## January 31, 2023
+
+### Android Places 2.0.0
+
+* Major version update for [Adobe Experience Platform Location Service](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-places-android).
```
Comment (swarna04) [inline src/pages/documentation/places/release-notes.md:7]: already -> now

Comment (swarna04) [reply src/pages/documentation/places/release-notes.md:7]: Can we also add info on the removal of an overloaded API with this release in the notes here.

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Review by calebk1 — APPROVED** [review]
After Swarna's changes are made, we'll be good! (Marking as approved so I don't have to re-review this a second time :p)

File: src/pages/documentation/release-notes/index.md [hunk 3]
```diff
@@ -161,15 +176,15 @@ Please note that the `registerDevice` API, similar to iOS, no longer provides a
...
 
 ### Android Edge 2.0.0
 
 * Major version update for [Edge Network](../edge-network/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edge-android).
 
 ### Android EdgeIdentity 2.0.0
 
-* Major version update for [Identity for Edge Network](../identity-for-edge-network/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edgeidentity-android).
+* Major version update for [Identity for Edge Network](../identity-for-edge-network/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-edgeidentity-android).
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:187]: These extensions - Consent, Optimize, Edge, Edge Identity etc were already open sourced with their 1.x releases so we stated so. Let's not update them.

File: src/pages/documentation/release-notes/index.md [hunk 4]
```diff
@@ -128,14 +128,29 @@ import com.adobe.marketing.mobile.target.TargetParameters;
 
 ### Android Messaging 2.0.0
 
-* Major version update for [Adobe Journey Optimizer](../adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is already available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
-* Adds support for Adobe Journey Optimizer powered in-app messages.
+* Major version update for [Adobe Journey Optimizer](../adobe-journey-optimizer/index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-messaging-android).
+
+### Android Places 2.0.0
+
+* Major version update for [Adobe Experience Platform Location Service](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-places-android).
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:135]: Let's fix the link here 
`[Adobe Experience Platform Location Service](../places/index.md)`

File: src/pages/documentation/release-notes/index.md [hunk 5]
```diff
@@ -128,14 +128,29 @@ import com.adobe.marketing.mobile.target.TargetParameters;
...
+
+Please note that the following improvements have been made in the current release:
+
+1. `getNearbyPointsOfInterest` API without the errorcallback has been removed.
```
Comment (swarna04) [inline src/pages/documentation/release-notes/index.md:139]: Suggest rephrase:
The `getNearbyPointsOfInterest` API without the errorCallback has been removed. Alternatively, use the below overloaded API which provides both successCallback and errorCallback: 
```java
public static void getNearbyPointsOfInterest(final Location location,
			final int limit,
			final AdobeCallback<List<PlacesPOI>> successCallback,
			final AdobeCallback<PlacesRequestError> errorCallback)
```

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Review by swarna04 — APPROVED** [review]

**Pull request merged**